cmd/admin/admin.go

```diff
@@ -0,0 +1,66 @@
+package main
```

Comment:
Hmm thanks for pointing these out. Honestly, my preferred solution would be to remove this subcommand entirely; it's hard for me to image it being super useful when someone could get this info from the playbook that told them to run `admin revoke-cert` in the first place.

Reply:
Instead of a list-reason, what if this information was in the usage for "revoke-cert"?  
Can revoke-cert -reason take a string from this set, so we're not handling the raw numbers in the CLI at all?

Reply:
I've removed list-reasons and changed revoke-cert -reason to take a string (and updated the usage to list the allowed strings).